.github/workflows/Lighthouse.yml

```
@@ -0,0 +1,20 @@
+name: Lighthouse
+on: [push, pull_request]
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)